### PR TITLE
docs: add project links to pyproject.toml

### PR DESCRIPTION
**Pull request by behnazh-w**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -106,10 +106,10 @@ test-docker = [
 ]
 
 [project.urls]
-Homepage = "https://my.project/"
-Changelog = "https://my.project/CHANGELOG"
-Documentation = "https://my.project/docs/"
-Issues = "https://my.project/issues"
+Homepage = "https://github.com/oracle/macaron"
+Changelog = "https://github.com/oracle/macaron/blob/main/CHANGELOG.md"
+Documentation = "https://oracle.github.io/macaron/"
```
Comment (tromai) [inline pyproject.toml:111]:
```suggestion
Documentation = "https://oracle.github.io/macaron"
```
To be consistent with other URLs in this section.

Comment (behnazh-w) [reply pyproject.toml:111]: `macaron` is a path parameter. That's why a `/` is automatically added. I can add `index.html` to the path.

Comment (tromai) [reply pyproject.toml:111]: Oh okay, if it's intended then I am all good with it. Thanks

**Review by tromai — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (behnazh-w) [reply pyproject.toml]: See commit [d0deb1d](https://github.com/oracle/macaron/pull/867/commits/d0deb1dfa696813029f64d1abef0136f5a862046).

**Pull request merged**